en/contributing/build.md

```diff
@@ -142,14 +148,14 @@ Also, you need to set these three environment variables:
 
 E.g. you can add the lines below to your .bashrc, (or .zshrc for zshell users, or .profile):
 
-```
+```sh
 export NDK_ROOT=$HOME/Android/android-ndk-r13
 export ANDROID_TOOLCHAIN_CMAKE=$HOME/Android/android-ndk-r13/build/cmake/android.toolchain.cmake
 export ANDROID_CMAKE_BIN=$HOME/Android/Sdk/cmake/3.6.3155560/bin/cmake
 ```
 
 Then you build for all Android architectures:
-```
+```sh
```

Comment:
Has this android build been re-tested/changed?

Reply:
I think so. I use it regularly.